pg_yregress/pg_yregress.c

```diff
@@ -69,6 +69,41 @@ static bool populate_ytest_from_fy_node(struct fy_document *fyd, struct fy_node
       }
     }
 
+    {
+      // Is the test meant to be ran on a different database?
+      struct fy_node *database = fy_node_mapping_lookup_by_string(test, STRLIT("database"));
+      y_test->database.base = NULL;
+
+      if (database != NULL) {
+        if (!fy_node_is_scalar(database)) {
+          fprintf(stderr, "database should be a string, got: %s",
+                  fy_emit_node_to_string(database, FYECF_DEFAULT));
+          return false;
+        }
+        if (!managed) {
+          fprintf(stderr, "Can't use `database` in unmanaged instances\n");
+          return false;
+        }
```

Comment:
Opted not to support managed instance because it may not necessarily be available at localhost and connecting to a different db would require additional password

Reply:
Perhaps worth mentioning in the documentation?

Reply:
The Caveats part of unmanged instance is updated to include `database` property, should be enough right ?

Reply:
I was thinking further about this. When we're given an unmanaged instance, we do have its host, but you are right that the credentials required may be different.

 I think it's safe to continue the way you suggest, and if there will be an interest in supporting this for unmanaged instances, we can enable this behavior and extend `database` key to support a object to configure more than just the database; or support URLs.